examples/predict_houseprice.py

```diff
@@ -0,0 +1,37 @@
+"""
+===================================================
+Price Predicting using pgmpy and BayesianModel
+===================================================
+For simplicity we will consider that the price of the house depends only on Area, Location, Furnishing,
+Crime Rate and Distance from the airport.  And also we will consider that all of these are discrete variables.
+"""
+from pgmpy.models import BayesianModel
+import pandas as pd
+import numpy as np
+data = pd.DataFrame(raw_data, columns=['A', 'C', 'D', 'L', 'F', 'P'])
```

Comment:
@ankurankan  I know that .I would take time to create data set.Because i have to be familiarize with pydataframe. so give me time .but if you would like to discard this then i will not work on this example please tell what should i do work on the creation of its data or leave it.